### PR TITLE
[feat] add sdxl-lite inference

### PR DESCRIPTION
**Pull request by heepengpeng**
* sdxl model export to lite 
*  sdxl model lite infer

### CODE REVIEW

File: examples/stable_diffusion_xl/inference/libs/tokenizer.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
+from typing import List, Union
+
+import numpy as np
+from gm.modules.embedders.open_clip.tokenizer import SimpleTokenizer
```
Comment (zhanghuiyao) [inline examples/stable_diffusion_xl/inference/libs/tokenizer.py:4]: 1、可以直接使用如下接口，
from gm.modules.embedders.open_clip import tokenize as openclip_tokenize
2、SimpleTokenizer/SimpleTokenizerWrapper 类名建议修改为 OpenCLIPTokenizer

File: examples/stable_diffusion_xl/inference/README.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+Please run `python sd_lite_infer.py -h` for details of command parameters.
+
+You can get images at "output/samples".
+
+## BenchMark
+
+| Model | task | batch size | image size | sample step | device | engine | time per image |
+| ----  | ---  | ---------- | ---------- | ----------- | ------ | ------ | -------------- |
+| sd_xl_base_1.0 | text2img | 1 | 1024*1024 | 40 | Ascend 910A | MindSpore | 17.20 s |
+| sd_xl_base_1.0 | text2img | 1 | 1024*1024 | 40 | Ascend 310P | MindSpore | 118 s |
```
Comment (zhanghuiyao) [inline examples/stable_diffusion_xl/inference/README.md:142]: time per image 建议先不写

File: examples/stable_diffusion_xl/inference/README.md [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+The MindIR file wil be generated in output/\[MODEL_NAME\]-\[TASK\]/.
+
+#### Export to MindSpore Lite Model
+
+Please use converter_lite command to convert MindSpore MindIR to MindSpore Lite model.
+
+Example:
+
+```shell script
+converter_lite --fmk=MINDIR  --saveType=MINDIR --optimize=ascend_oriented --modelFile ./output/\[MODEL_NAME\]-\[TASK\]/data_prepare_graph.mindir --outputFile=./output/\[MODEL_NAME\]-\[TASK\]/data_prepare_graph_lite.mindir
```
Comment (zhanghuiyao) [inline examples/stable_diffusion_xl/inference/README.md:116]: 增加sd_lite.cfg传参

File: examples/stable_diffusion_xl/inference/README.md [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+````
+Note:
+Lite model name is end with _lite.
+
+After all models conversion, please use `sd_lite_infer.py` to generate images.
+
+### Lite Inference
+
+Run `sd_lite_infer.py` to generate images for the prompt of your interest.
```
Comment (zhanghuiyao) [inline examples/stable_diffusion_xl/inference/README.md:125]: 1、后续可以增加 910B 推理适配
2、后续可以增加 streamlit 交互式推理
3、后续可以增加 refiner、pipeline 推理

File: examples/stable_diffusion_xl/inference/libs/infer_engine/export_modules.py [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+    def vae_decode(self, x):
+        y = self.vae.decode(ms.ops.div(x, self.scale_factor))
+        y = ops.clip_by_value((y + 1.0) / 2.0, clip_value_min=0.0, clip_value_max=1.0)
+        return y
+
+    def construct(self, latents):
+        return self.vae_decode(latents)
+
+
+class Text2ImgDataPrepare(DataPrepare):
```
Comment (zhanghuiyao) [inline examples/stable_diffusion_xl/inference/libs/infer_engine/export_modules.py:131]: Text2ImgDataPrepare -> Embedder

Comment (heepengpeng) [reply examples/stable_diffusion_xl/inference/libs/infer_engine/export_modules.py]: 已修改

File: examples/stable_diffusion_xl/inference/README.md (hunk 3 above)
Comment (heepengpeng) [reply examples/stable_diffusion_xl/inference/README.md]: 已修改

**Review by SamitHuang — APPROVED** [review]
What are the key changes compared to `stable_diffusion_v2/inference`? 

The overall pipeline seems the same, and model components are already switchable in `stable_diffusion_v2/inference`. It is possible to share the same inference framework in the future?

**Review by zhanghuiyao — APPROVED** [review]

**Pull request merged**